docs/sources/installation/docker.md

```diff
@@ -58,9 +58,9 @@ docker run -d -p 3000:3000 --name grafana grafana/grafana:6.5.0
 
 ### Run the Grafana master branch
 
-For every successful build of the master branch, we update the `grafana/grafana:master` and `grafana/grafana:master-ubuntu` tags. Additionally, two new tags are created, `grafana/grafana-dev:master-<commit hash>` and `grafana/grafana-dev:master-<commit hash>-ubuntu`, which includes the hash of the Git commit that was built. Use these to get access to the latest master builds of Grafana.
+For every successful build of the master branch, we update the `grafana/grafana:master` and `grafana/grafana:master-ubuntu` tags. Additionally, two new tags are created, `grafana/grafana-dev:master-<build ID>pre` and `grafana/grafana-dev:master-<build ID>pre-ubuntu`, which include the ID of the corresponding CI build. Use these to get access to the latest master builds of Grafana.
```

Comment:
Maybe mention how to find the build ID?

Reply:
@sakjur I'm not sure how to easily locate the CircleCI build ID, which is technically the workflow ID. I can't see it available in the UI.

Reply:
It's also worth considering that we will change master builds over to Drone soon, if all goes to plan.

Reply:
It's the first part of the UUID in the URL, but other than that idk https://app.circleci.com/pipelines/github/grafana/grafana/15051/workflows/648d9af7-4f64-4aa0-97af-433ca2ea4226/

Also, the tag seems to be `<next target release>-<build ID>pre`, not `master-<build ID>pre`.

`grafana/grafana-dev:master-648d9af7pre` isn't found for me, but `grafana/grafana-dev:7.3.0-648d9af7pre` is available.

Reply:
@sakjur Oh yeah, you're right about the master- prefix. Will fix.